### ISSUE TITLE
Trainer: Support LR Schedulers

### ISSUE THREAD

**Issue by haroldrubio** [haroldrubio/PytorchTrainer#10]

**Comment by haroldrubio** [comment]
As a scheduler is an option, but not a necessity for a model to learn, it will be supported by any hyperparameter optimization for the scheduler must be done outside of the trainer object as it will not be supported in the class' hyperparameter optimization function

**Comment by haroldrubio** [comment]
Scheduler has been implemented
